homeassistant/components/uptime_kuma/__init__.py

```diff
@@ -0,0 +1,60 @@
+"""The Uptime Kuma integration."""
+from datetime import timedelta
+import logging
+
+import async_timeout
+from uptime_kuma_monitor import UptimeKumaMonitor
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import update_coordinator
+
+from .const import DOMAIN
+
+PLATFORMS = ["binary_sensor"]
```

Comment:
Not compulsory, but I like to use the constant:
`from homeassistant.components.binary_sensor import DOMAIN as BINARY_SENSOR_DOMAIN`
And then `PLATFORMS = [BINARY_SENSOR_DOMAIN]`